### PR TITLE
ibu mgmt: add missing dpa test

### PR DESCRIPTION
**Pull request by trewest**
Add missing DPA test to the negative test suite. Also update IBU clean up to be contained in one suite level AfterEach container.

### CODE REVIEW

File: tests/lca/imagebasedupgrade/mgmt/negative/negative_suite_test.go [hunk 1]
```diff
@@ -35,6 +38,48 @@ var _ = BeforeSuite(func() {
...
+		Expect(err).NotTo(HaveOccurred(), "error waiting for idle stage to complete")
+	}
+
+	Eventually(func() (bool, error) {
+		ibu.Object, err = ibu.Get()
+		if err != nil {
+			return false, err
+		}
+
+		return len(ibu.Object.Status.Conditions) == 1 &&
```
Comment (achuzhoy) [inline tests/lca/imagebasedupgrade/mgmt/negative/negative_suite_test.go:74]: why do we check for having a single condition?

Comment (trewest) [reply tests/lca/imagebasedupgrade/mgmt/negative/negative_suite_test.go:74]: Because when the IBU is in a proper Idle state, there is only one condition. e.g.
```
$ oc get ibu upgrade -o yaml | yq e .status -  
conditions:
  - lastTransitionTime: "2024-06-21T15:28:39Z"
    message: Idle
    observedGeneration: 197
    reason: Idle
    status: "True"
    type: Idle
observedGeneration: 197
validNextStages:
  - Prep
```

**Review by achuzhoy — APPROVED** [review]
